lib/log.ex

```diff
@@ -0,0 +1,52 @@
+defmodule KvStorageRedis.Log do
+  @levels %{
+    emergency: 8,
+    alert: 7,
+    critical: 6,
+    error: 5,
+    warning: 4,
+    notice: 3,
+    info: 2,
+    debug: 1
+  }
+
+  @conf_level get_in(Application.get_env(:kv_storage_redis, __MODULE__), [:level])
+  @conf_priority get_in(@levels, [@conf_level])
```

Comment:
:)

`get_in/2` is surely cool, but in configs we usually stick to strictly using keywords and `Keyword.get_in/2` should suffice in the former case. `@levels` is already the map, and `Map.fetch!/2` would be better here to make sure we indeed got a value.

А еще (и`credo` сказало бы об этом), `Application.get_in/3` используют в рантайме; тут нужен `compile_env/3`. Ну и default.
```suggestion
  @config Application.compile_env(:kv_storage_redis, __MODULE__, [])
  @conf_level Keyword.get_in(@config, :level, :info)
  @conf_priority Map.fetch!(@levels, @conf_level)
```